libs/hpc-ui/src/lib/components/form-fields/input-entry.tsx

```diff
@@ -0,0 +1,142 @@
+        }}
+      >
+        <div style={{ color: THEME.colors.primary.normal }}>
+          <span style={{ fontWeight: 800 }}>Change: </span>
+          <span>
+            {info.kind === forms.InputEntryKindsEnum.NEW && 'New Value'}
+            {info.kind === forms.InputEntryKindsEnum.REVISED && 'Revised Value'}
+            {info.kind === forms.InputEntryKindsEnum.DELETED && 'Blank Value'}
+            {info.kind === forms.InputEntryKindsEnum.UNMATCHED &&
+              'External Value[Unmatched]'}
+          </span>
+        </div>
+        {info.category === forms.InputEntryCategoriesEnum.ACTIVE_FLOW && (
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              marginBottom: 6,
+            }}
+          >
+            <div
+              style={{
+                color: '#357A38',
+              }}
+            >
+              <IconButton
+                aria-label="accept"
+                sx={{
+                  color: THEME.colors.greenLight,
+                  '&:hover': {
+                    backgroundColor: THEME.colors.greenLight,
+                    color: '#fff',
+                  },
+                  height: 20,
+                  width: 20,
+                }}
+                onClick={handleSetValue}
+              >
+                <Check sx={{ height: 16, width: 16 }} />
+              </IconButton>
+              Accept
+            </div>
+            <div
+              style={{
+                color: '#FE3A40',
+              }}
+            >
+              <IconButton
+                aria-label="reject"
+                sx={{
+                  color: THEME.colors.secondary.normal,
+                  '&:hover': {
+                    backgroundColor: THEME.colors.secondary.normal,
+                    color: '#fff',
+                  },
+                  height: 20,
+                  width: 20,
+                }}
+                onClick={handleRejectValue}
+              >
+                <Close sx={{ height: 16, width: 16 }} />
+              </IconButton>
+              Reject
+            </div>
+          </div>
+        )}
+      </div>
+      <div>
+        {info.kind === forms.InputEntryKindsEnum.DELETED ? (
+          '[Blank]'
+        ) : (
+          <>
+            {typeof info.value === 'string' || typeof info.value === 'number'
+              ? info.value
+              : Array.isArray(info.value)
+              ? info.value
+                  .map((value) => {
+                    let result = value.displayLabel;
+                    if (value.isInferred) {
+                      result = result + '(I)';
+                    }
+                    if (value.isTransferred) {
+                      result = result + '(T)';
+                    }
+                    return result;
+                  })
+                  .join('\n')
+              : info.value?.displayLabel}
+          </>
```

Comment:
Remove fragment, there is only one child (unless you want to wrap it around a container like a `<div />` if needed)
```suggestion
            typeof info.value === 'string' || typeof info.value === 'number'
              ? info.value
              : Array.isArray(info.value)
              ? info.value
                  .map((value) => {
                    let result = value.displayLabel;
                    if (value.isInferred) {
                      result = result + '(I)';
                    }
                    if (value.isTransferred) {
                      result = result + '(T)';
                    }
                    return result;
                  })
                  .join('\n')
              : info.value?.displayLabel
```